massgov/mayflower
# DP-16690 Swap wording of less to fewer

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description
I swapped the wording in the Featured Mosaic Item to be "Fewer" instead of "Less".
Also I adjusted the JS used in the Image Link List organism which is used in several places.

Patterns:
http://localhost:3000/?p=organisms-featured-item-mosaic
http://localhost:3000/?p=organisms-image-link-list

Example pages (view mobile size):
http://mass.local/orgs/group-insurance-commission
http://mass.local/orgs/office-of-the-governor

## Related Issue / Ticket

- https://jira.mass.gov/browse/DP-16690


## Steps to Test


1. Visit this pattern http://localhost:3000/?p=organisms-featured-item-mosaic
2. View as mobile size
3. Verify button text now says "Fewer featured items"
4. Visit http://localhost:3000/?p=organisms-image-link-list on mobile and check the same for organizations
5. Check the http://mass.local/orgs/group-insurance-commission and http://mass.local/orgs/office-of-the-governor pages too

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


*

#### @TODO


*

#### Today I learned...
